### PR TITLE
Add ability to set app as launcher

### PR DESCRIPTION
**Pull request by mueller-ma**
Requested in https://community.openhab.org/t/android-app-org-openhab-habdroid-home-screen-launcher-kiosk-mode/136499

Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

File: mobile/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -57,6 +57,7 @@
     <string name="settings_accent_color">Accent color</string>
     <string name="settings_select_a_color">Select a color</string>
     <string name="settings_openhab_fullscreen">Fullscreen</string>
+    <string name="settings_set_as_launcher">Set as launcher</string>
```
[the remark this answers is not in the document]
Comment (mueller-ma) [reply mobile/src/main/res/values/strings.xml]: Yes, that's better

**Comment by imaginator** [comment]
I see this mentioned in 2.20.13-beta but could it be that the code didn't make it into the beta release at https://github.com/openhab/openhab-android/releases/tag/2.20.13-beta ?

**Comment by mueller-ma** [comment]
The code in 2.20.13 is there, but crashes. The fix will be in 2.20.14.

**Comment by imaginator** [comment]
Indeed it is. I was expecting it to just show up in the list of launchers,
then I found the switch (which as you mentioned, seems to cause a crash
(submitted crash logs if they are of any use)).

**Comment by joshuakoh1** [comment]
@mueller-ma is there a way to disable the pulldown menu in this kiosk mode?

**Pull request merged**